iogt_users/templates/profile.html

```diff
@@ -11,35 +11,35 @@
 
 {% block allauth_content %}
     {% if user.username %}
-        <h1 class="title profile__title"> Hey, {{ user.username }}
-            <span
-                    class="profile__icon">{{ user.username | first | upper }}</span>
+        <h1 class="title profile__title">{% trans "Hey" %}, {% trans user.username %}
```

Comment:
I don't think the username should be translated?

Reply:
Would it make sense to use a blocktranslate tag with a username placeholder here?